py/runtime0.h

```diff
@@ -51,55 +51,50 @@ typedef enum {
     MP_UNARY_OP_NOT,
 
     // Following ops cannot appear in the bytecode
-    MP_UNARY_OP_NON_BYTECODE,
+    MP_UNARY_OP_LAST_BYTECODE = MP_UNARY_OP_NOT,
```

Comment:
A common pattern is to add new items at the end of a list. And then someone may overlook the need to patch yet another line when doing so (or just have to patch 2 lines because of that). That's why I did it like it was. And yep, initially I called it "LAST" too, but as it's not really last, I renamed it to "NON" before submitting. Since then I remembered I better term for such cases - "LIMIT".

(Explaining why it was done like it was done. Not calling to get it back, though in other cases, "patch one line instead of 2" is a good criterion to use).